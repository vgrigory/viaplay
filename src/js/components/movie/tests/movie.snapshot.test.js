import React from 'react';
import Movie from '../movie';
import renderer from 'react-test-renderer';

it('snapsot test for Movie component', () => {
    let movie = {
        filmId: 'titanic-1997'
    };

    const tree = renderer.create(
        <Movie params={movie} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});