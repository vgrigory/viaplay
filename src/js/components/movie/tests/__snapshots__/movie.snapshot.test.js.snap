exports[`test snapsot test for Movie component 1`] = `
<div
  className="noMovieNoCry">
  Ooops, we\'ll add this movie ASAP. Meanwhile try 
  <a
    href="/film/titanic-1997">
    Titanic
  </a>
  .
</div>
`;
